lib/PreconditionRunner.pm

```diff
@@ -95,12 +96,53 @@ sub write_num_passes
 sub run_to_steady_state($)
 {
     my $self = shift;
-    
     my %args = @_;
 
     my $msg_prefix = $args{'msg_prefix'};
-    my $output_dir = $args{'output_dir'};
     my $test_ref = $args{'test_ref'};
+    my $output_dir = $args{'output_dir'};
+
+    # preconditioner.exe does not support multiple files
```

Comment:
This is one area I'm not totally happy with yet.  I can't run multiple copies of preconditioner.exe, because I need them all to keep running until they all reach steady state.  I put the functionality in precondition runner because this is conceptually a precondition step (to use SNIA's terminology).

One level up is in Target.pm.    I want to give it some more thought, but I'm open to putting it there.